### ISSUE TITLE
Attempting to set an AKICK on an user with privileges returns the wrong error message

### ISSUE THREAD

**Issue by Zarthus** [atheme/atheme#544]
When attempting to set an AKICK on an opper user (had nobody to test with, I had +Ffr) you receive the following error message:

```
(14:31:37) -ChanServ- Zarthus already has flags +Ffr on #er9w0jwfe9j0ewf'
```

I presume an error message along the likes of was intended:
```
(14:31:37) -ChanServ- Zarthus is an op on #er9w0jwfe9j0ewf', and cannot be akicked.
```

The following command was used:
```
/cs akick #er9w0jwfe9j0ewf' add zarthus hi
```

Note that when setting an akick on hostmasks, the akick just persists. but that's pretty hard to map out accurately anyways.

Tested on freenode (7.2.5) and another older version of atheme, it might already have been fixed.

**Comment by loop-de-r00t** [comment]
This is also the case with atheme 7.2.8

**Comment by Zarthus** [comment]
@loop-de-r00t - a bit off topic, but it's worth upgrading to 7.2.9

**Comment by loop-de-r00t** [comment]
No one on the network has the impersonation privilege

**Comment by GLolol** [comment]
If I recall correctly, it's the `+r` flag in particular that gives users immunity from akicks. Perhaps a more concise message could be "$user has the +r flag in #channel and cannot be akicked".

**Comment by ilbelkyr** [comment]
@GLolol `+e`, actually, but the real issue here is that akicks are just a `+b` chanacs with some metadata set on them, so services complain they can't add a new entry with just `+b` since there already is an entry with other flags.

I do wonder if it might be worth separating akicks from chanacs; this would also help with things like #548.

**Comment by GLolol** [comment]
(`+e` is true if `chanserv/unban_self` is loaded, otherwise it's `+r` iirc.)

**Comment by ilbelkyr** [comment]
No, `+e` is exemption from bans and akicks, including the ability to unban yourself specifically; `+r` is the ability to use quiet/unquiet/ban/unban/kick on other people, plus (together with `+f`) to set akicks. (`+e` used to be part of `+r` but was split off so you could exempt people from akicks without giving them the ability to set their own bans etc.)

**Comment by ilbelkyr** [comment]
This will be fixed as part of #676.
